kasa/iot/modules/lighteffectmodule.py

```diff
@@ -0,0 +1,97 @@
+"""Module for light effects."""
+
+from __future__ import annotations
+
+from ...modules.lighteffectmodule import LightEffectModule as BaseLightEffectModule
```

Comment:
Perhaps we should call the "base modules" interfaces or module interfaces? It would avoid having to import these using aliasing, and make it clear that they are just interfaces implemented by modules?

```
from ...interfaces import LightEffectInterface

class LightEffectModule(IotModule, LightEffectInterface):
```
or maybe even
```
from ...interfaces import LightEffect

class LightEffectModule(IotModule, LightEffect)
```
?

Reply:
With the previous implementation of `get_module` the name needed to match the name of the submodule.  With the new typing approach this is no longer the case and I have renamed as per your suggestion.